.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '34 17 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp' ]
```

Comment:
@eyitemi-paystack  So the problem with CodeQL is that its not scanning our files correctly as we use ObjC and Swift. I have spoke to Jocelyn about this and she said if we are using Sonar to scan this repo which we are already for security vulnerabilities then it should be fine. Ill confirm with her and maybe if you can also check with her 😊

Reply:
@eyitemi-paystack  this is the link to Sonar Cloud for this repo for context, https://sonarcloud.io/summary/new_code?id=PaystackHQ_paystack-ios

Reply:
@eyitemi-paystack Awesome, thats what I spoke to Jocelyn about so if the security team is happy, we will be using Sonar for all mobile repos. Can we then close this PR?  cc: @ahmed-paystack

Reply:
Yes, please. Let's close this PR.